src/openrct2/entity/Staff.cpp

```diff
@@ -2364,7 +2366,8 @@ bool Staff::UpdateFixingFixStationStart(bool firstRun, const Ride& ride)
 {
     if (!firstRun)
     {
-        if (ride.GetRideTypeDescriptor().HasFlag(RIDE_TYPE_FLAG_HAS_SINGLE_PIECE_STATION | RIDE_TYPE_FLAG_HAS_NO_TRACK))
+        if (ride.GetRideTypeDescriptor().HasFlag(RIDE_TYPE_FLAG_HAS_SINGLE_PIECE_STATION) &&
```

Comment:
This might be less performant than it used to be when the flag was used. 
I could avoid those by removing `RIDE_TYPE_FLAG_HAS_TRACK` and keeping `RIDE_TYPE_FLAG_HAS_NO_TRACK` instead, but I thought that change could be less intuitive.

Reply:
Well its more logical to signal that something is present not the other way around to be honest, this shouldn't really have any impact performance wise, compilers optimizations are quite smart for such trivial cases.